cnf-tests/testsuites/e2esuite/test_suite_test.go

```diff
@@ -93,47 +94,49 @@ func TestTest(t *testing.T) {
 }
 
 var _ = BeforeSuite(func() {
-	Expect(testclient.Client).NotTo(BeNil())
-	// create test namespace
-	ns := &corev1.Namespace{
-		ObjectMeta: metav1.ObjectMeta{
-			Name: namespaces.NamespaceTesting,
-		},
-	}
-	_, err := testclient.Client.Namespaces().Create(context.Background(), ns, metav1.CreateOptions{})
-	Expect(err).ToNot(HaveOccurred())
+	if os.Getenv("SKIP_TEST_NAMESPACES_CREATION") != "true" {
```

Comment:
Can we read this in init and set a flag?

Reply:
I set it as a flag and then realized that I'm breaking the run-functests.sh script that way. The same script is used to run the validation, configuration and functests suites but we only have that flag in the functests suite so the other suites are failing for that. I can add a check to the run-functests.sh script which suite we are running, but if we run two I will have to slpit the command and I think that's ugly. I want to leave it this way unless you have a cleaner idea.